OCPBUGS#44659-13: Added important note for disabling IPsec for 4.13 a… (#85581)
openshift/openshift-docs: modules/nw-ovn-ipsec-disable.adoc

As a cluster administrator, you can disable IPsec encryption only if you enabled IPsec after cluster installation.

[IMPORTANT]
====

> To avoid issues with your installed cluster after you disable IPsec, ensure that you delete the associated IPsec daemonsets pods.

Reviewer: Does a stronger phrasing make sense for this note?
```suggestion
After disabling IPsec, you must delete the associated IPsec daemonsets pods.
```

  -p='[{"op":"remove", "path":"/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig"}]'
----

. To find the name of the OVN-Kubernetes data plane pod that exists on the `master` node in your cluster, enter the following command:
+
[source,terminal]
----
$ oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-master
----
+
.Example output
[source,terminal]
----
ovnkube-master-5xqbf                      8/8     Running   0              28m
----

. Verify that the `master` node in your cluster has IPsec disabled by entering the following command. The command output must state `false` to indicate that the node has IPsec disabled.
+
[source,terminal]
----
$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-XXXXX \// <1>
  ovn-nbctl --no-leader-only get nb_global . ipsec 
----

> <1> Replace `XXXXX` with the random sequence of letters for the data plane pod from the previous step.

Reviewer: Should this use the [standard convention](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values) for `<user_replaceable_values>`?

----

. To find the name of the OVN-Kubernetes data plane pod that exists on the `master` node in your cluster, enter the following command:
+
[source,terminal]
----
$ oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-master
----
+
.Example output
[source,terminal]
----
ovnkube-master-5xqbf                      8/8     Running   0              28m
...
----

. Verify that the `master` node in your cluster has IPsec disabled by entering the following command. The command output must state `false` to indicate that the node has IPsec disabled.
+
[source,terminal]
----
$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<pod_number_sequence> \// <1>
  ovn-nbctl --no-leader-only get nb_global . ipsec 
----
<1> Replace `<pod_number_sequence>` with the random sequence of letters, such as `5xqbf`, for the data plane pod from the previous step.

. To remove the IPsec `ovn-ipsec` daemonset pod from the `openshift-ovn-kubernetes` namespace on the node, enter the following command:
+
[source,terminal]
----
$ oc delete daemonset ovn-ipsec -n openshift-ovn-kubernetes <1>
----
<1> The `ovn-ipsec` daemonset configures IPsec connections for east-west traffic on the node.

> . Verify that the `ovn-ipsec` daemonset pod was removed from the `master` node in your cluster by entering the following command. If the command output does not list the pod, the removal operation is successful.

Reviewer 2: The `ovn-ipsec` daemonset pod exists in all nodes, not only on the `master` node.

Author: :heavy_check_mark:

+
[source,terminal]
----

> $ oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-master

Reviewer 2: `$ oc get pods -n openshift-ovn-kubernetes -l=app=ovn-ipsec`
